pkg/controllers/provisioning/scheduling/nodeclaim.go

```diff
@@ -256,6 +265,13 @@ func filterInstanceTypesByRequirements(instanceTypes []*cloudprovider.InstanceTy
 			results.remaining = append(results.remaining, it)
 		}
 	}
+	if requirements.HasMinValues() {
+		// We do not need minimum number of instanceTypes to satisfy all the minimum requirements here since we always send
+		// 100 instancetypes to launch path
+		results.requirementIncompatibleWithMinValues, _ = IncompatibleReqAcrossInstanceTypes(requirements, results.remaining)
+	}
+	// If minValues is NOT met for any of the requirement across InstanceTypes, then return empty InstanceTypeOptions as we cannot launch with the remaining InstanceTypes.
+	results.remaining = lo.Ternary(len(results.requirementIncompatibleWithMinValues) > 0, []*cloudprovider.InstanceType{}, results.remaining)
```

Comment:
Sanity check: When we make the `ToNodeClaim()` call and we make the instance type requirement do we carry the minValues requirement for instance types with us? If not, we need to add that or else we are going to be generating an instance type requirement without the minValues that have been specified